urbanairship-location-react-native/android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.urbanairship.reactnative.location">
```

Comment:
I am pretty sure we need to add the location permission to the manifest, i dont think our module includes it by default but it should start.